stone-java-gradle-plugin/src/main/kotlin/com/dropbox/stone/java/StoneTask.kt

```diff
@@ -6,17 +6,13 @@ import org.gradle.api.file.DirectoryProperty
 import org.gradle.api.file.RegularFileProperty
 import org.gradle.api.provider.ListProperty
 import org.gradle.api.provider.Property
-import org.gradle.api.tasks.Input
-import org.gradle.api.tasks.InputDirectory
-import org.gradle.api.tasks.InputFile
-import org.gradle.api.tasks.OutputDirectory
-import org.gradle.api.tasks.TaskAction
-import org.gradle.api.tasks.Optional
+import org.gradle.api.tasks.*
 import org.gradle.process.ExecOperations
 import java.io.File
 import java.io.FileOutputStream
 import javax.inject.Inject
 
+@CacheableTask
```

Comment:
With help of @devPalacio - marks this Task as cacheable which should increase build speeds